packages/staking/src/features/delegation-card/DelegationCard.tsx

```diff
@@ -118,7 +114,13 @@ export const DelegationCard = ({
         data-testid="delegation-info-card"
       >
         <div className={styles.chart} data-testid="delegation-chart">
-          <PieChart data={data} nameKey="name" valueKey="percentage" colors={colorSet} />
+          <PieChart
+            data={data}
+            nameKey="name"
+            valueKey="percentage"
+            colors={colorSet}
+            tooltip={DelegationTooltip as any}
```

Comment:
Rather than casting to `any` which basically breaks off the DelegationTooltip interface from the real interface expacted by the Recharts component, I'd consider declaring an easier to follow interface within the `PieChart` component. As far as I tried an interface like this should do and actually plays well with Recharts without requiring any re-casting, while being a lot simpler to follow:

```typescript
interface TooltipContentRendererProps<T> {
  active?: boolean;
  payload?: { name?: string; payload?: T }[];
}
interface TooltipProps<T> {
  content:
    | ReactElement
    | ((props: TooltipContentRendererProps<T>) => ReactNode);
}
``` 

```suggestion
            tooltip={DelegationTooltip}
```

Reply:
Fixed on c70445d